### PR TITLE
Pin hmpps-auth to the latest working version to get builds passing

### PR DESCRIPTION
**Pull request by devkokov**

### CODE REVIEW

**Review by LucasCairns — APPROVED** [review]
LGTM 👍

File: docker-compose.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ services:
       - /var/lib/postgresql
 
   hmpps-auth:
-    image: quay.io/hmpps/hmpps-auth:latest
+    image: quay.io/hmpps/hmpps-auth::2024-05-09.23133.61d6975
```
Comment (LucasCairns) [inline docker-compose.yml:20]: `quay.io/hmpps/hmpps-auth::2024-05-09.23133.61d6975`
appears to have an extra `:`

Comment (devkokov) [reply docker-compose.yml:20]: oops. thanks 😄 

```suggestion
    image: quay.io/hmpps/hmpps-auth:2024-05-09.23133.61d6975
```

**Review by LucasCairns — APPROVED** [review]
👍

**Pull request merged**